install_sct

```diff
@@ -698,9 +698,7 @@ else
   # Download data
   print info "Installing data..."
   run mkdir -p "$SCT_DIR/$DATA_DIR"
-  for data in PAM50 deepseg_sc_models deepseg_gm_models deepseg_lesion_models deepreg_models PAM50_normalized_metrics; do
-    run sct_download_data -d "$data"
-  done
+  python/envs/venv_sct/bin/python -c 'import spinalcordtoolbox.download; spinalcordtoolbox.download.install_default_datasets()'
```

Comment:
Hrm... this doesn't seem to have the same `logging` output as `sct_download_data`. I'll have to look into that...

Reply:
Fixed in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4484/commits/6401119a0f71c5fba20e576031ca0ea82a463c4c.